runtime/src/bank.rs

```diff
@@ -5270,6 +5271,24 @@ impl Bank {
         );
         load_time.stop();
 
+        let preexecution_account_states: Option<HashMap<Pubkey, AccountSharedData>> = self
```

Comment:
Not sure that runtime needs another .cloned().collect(). These will do tons of allocations and make runtime slower than it already is